ios/Exponent.xcodeproj/xcshareddata/xcschemes/Expo Go (unversioned).xcscheme

```diff
@@ -31,7 +31,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
Are you sure about this? 😅

Reply:
good catch! sorry for accidentally committed this change 😅